packages/base/src/commands.ts

```diff
@@ -346,299 +320,48 @@ export function addCommands(
 
   commands.addCommand(CommandIDs.buffer, {
     label: trans.__('Buffer'),
-    isEnabled: () => {
-      const selectedLayer = getSingleSelectedLayer(tracker);
-      if (!selectedLayer) {
-        return false;
-      }
-      return ['VectorLayer', 'ShapefileLayer'].includes(selectedLayer.type);
-    },
+    isEnabled: () => isLayerTypeSupported(tracker),
```

Comment:
The API of this function is unclear.

May I suggest something like

```suggestion
    isEnabled: () => selectedLayerIsOfType(['VectorLayer', 'ShapefileLayer'], tracker),
```

Reply:
Done 🚀